core/Listener/BeforeTemplateRenderedListener.php

```diff
@@ -53,8 +55,8 @@ public function handle(Event $event): void {
 
 
 				// If installed and background job is set to ajax, add dedicated script
-				if ($this->config->getAppValue('core', 'backgroundjobs_mode', 'ajax') == 'ajax') {
-					Util::addScript('core', 'backgroundjobs');
+				if ($this->appConfig->getAppValueString('backgroundjobs_mode', 'ajax') === 'ajax') {
```

Comment:
```suggestion
				if ($this->appConfig->getValueString('backgroundjobs_mode', 'ajax') === 'ajax') {
```

If you change to the other IAppConfig

Reply:
done